consensus/safety-rules/src/configurable_validator_signer.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
```

Comment:
this isn't required unless it isn't in main.rs and lib.rs files.

Reply:
Ooh, thanks! Copy pasta 😄